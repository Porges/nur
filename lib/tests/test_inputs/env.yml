version: 1.0

default:
  after: [override, outer, inner]

override:
  env: 
    NAME: outer

  run:
    - sh: echo $NAME
      env:
        NAME: inner-override

outer:
  env: 
    NAME: outer

  run:
    - sh: echo $NAME

inner:
  run:
    - sh: echo $NAME
      env: 
        NAME: inner
