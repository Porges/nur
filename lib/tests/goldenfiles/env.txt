--- stdout ---
   inner │ — Started task ‘inner’
   outer │ — Started task ‘outer’
override │ — Started task ‘override’
   inner │ inner
         │ — Task ‘inner’ completed
   outer │ outer
         │ — Task ‘outer’ completed
override │ inner-override
         │ — Task ‘override’ completed
 default │ — Started task ‘default’
         │ — Task ‘default’ completed
--- stderr ---
